src/main/java/org/folio/bulkops/service/BulkOperationService.java

```diff
@@ -483,6 +482,20 @@ private UnifiedTable buildPreviewFromCsvFile(String pathToFile, Class<? extends
     return table;
   }
 
+  private void processNoteFields(UnifiedTable table, Class<? extends BulkOperationsEntity> clazz) {
+    table.getHeader().forEach(cell -> {
+      if ("Administrative Notes".equalsIgnoreCase(cell.getValue())) {
```

Comment:
"Administrative Notes" - make this as constant.

Reply:
Good catch, fixed